repo_policy_compliance/check.py

```diff
@@ -366,16 +368,16 @@ def execute_job(
         Whether the workflow run has been approved for the commit SHA.
     """
     repository = github_client.get_repo(repository_name)
-    maintain_logins = {
+    push_logins = {
         collaborator["login"]
         for collaborator in get_collaborators(
-            repository=repository, permission="maintain", affiliation="all"
+            repository=repository, permission="push", affiliation="all"
```

Comment:
Just to be sure: when this line query for collaborators with push permission will return the ones that have this and above as well, right? Or only push?

Reply:
correct, it essentially returns anyone that has the push permission which is included in the maintain and admin permission